src/data/multiset/sort.lean

```diff
@@ -50,7 +49,7 @@ list.merge_sort_singleton r a
 
 end sort
 
-instance [has_repr α] : has_repr (multiset α) :=
-⟨λ s, "{" ++ string.intercalate ", " ((s.map repr).sort (≤)) ++ "}"⟩
+meta instance [has_repr α] : has_repr (multiset α) :=
```

Comment:
Do you think it's worth adding a reference to gh-18166, or is it not a good enough idea for that?

```suggestion
-- TODO: use a sort order if available, gh-18166
meta instance [has_repr α] : has_repr (multiset α) :=
```